src/components/Form/Form.tsx

```diff
@@ -0,0 +1,52 @@
+import {
+	Form,
+	StyledDiv,
+	StyledFieldset,
+	StyledInput,
+	StyledLegend,
+} from "./Form.Styled.ts";
+import { useState } from "react";
+import NavLinkButton from "../Button/NavLinkButton.tsx";
+
+function LoginForm() {
+	const [password, setPassword] = useState("");
+	const [name, setName] = useState("");
+
+	return (
+		<Form
+			onSubmit={(e) => {
+				e.preventDefault();
+			}}
```

Comment:
Przenieśmy to wyżej, i zróbmy redirect usera tylko wtedy gdy email ma długośc, format oraz jest wypełniony wraz z wypełnionym hasłem min. 6 znaków. Jeżeli są błędy, to wyświetlmy je jakoś :P